.travis.yml

```diff
@@ -1,21 +1,19 @@
 sudo: false
 language: python
 python:
-  - '3.5'
-  - '3.5-dev'
+  - '3.6'
 matrix:
   allow_failures:
-    - python: "3.5-dev"
+    - python: "3.6"
   include:
     # This is the ltd-mason documentation deployment build
     - python: "3.5"
       env: LTD_MASON_BUILD=true
 install:
   - pip install -r requirements.txt
-  - pip install ltd-mason==0.2.0rc0
+  - pip install ltd-mason==0.2.2
```

Comment:
Yeah, though goal was to separate this pure-Travis dependency from application dependencies (i.e., ltd-mason should only be installed under Travis).